### ISSUE TITLE
Only working for creator of object

### ISSUE THREAD

**Issue by ChryslerSupport** [bohua/nprinting-sense-on-demand#4]
I LOVE THIS EXTENSION, IT WILL BE HUGE FOR US!!!

I have ran into an issue where i can only get the on demand report to run for the person that created the extension.  Anyone else that clicks the button doesn't get any response.
I have...
1.  Made sure the user is in the NPrinting dashboard with roles of User and NewsStand User
2.  Made sure to allow the certificate error to the NPrinting server.

What else should I try?  Is this a known issue?

Once again, thanks for this extension!  It could be a game changer for us in Chrysler and finally get Tableau to go away completely.

Thanks
Larry

**Comment by bohua** [comment]
Hi Larry, 
Thank you for your input and feedback! That's huge for us to keep on creating out-of-box extensions! :)

Since you have been successfully running this extension with certain user, which implies the connections and configurations are all handled correctly, which is a good start...

It's not a known issue for us so far. The first thing comes into my mind would still be user credentials. I don't have very good clue for details without investigation. I will suggest:
There must an error thrown in web browser console. If you could enable dev-tool (F12) on your browser (Google chrome would be the best). Refresh the page if there's no error captured. And press the extension button and copy the error in dev-tool console, and paste it back to me. 

Then we will be able to pinpoint the root cause from there. 👍 

Hope it helps and please feel free to contact! 

I am working from Danmark so the timezone is CET (+1),  we can host a remote meeting to have the investigation smoothly! My business mailbox is: bl@s-cubed.dk

Thanks!
//Bohua

**Comment by ChryslerSupport** [comment]
I am sending you the screen of the console when I click the button both before accepting the certificate error and after.  Both have the same result but different error messages.

Thanks
Larry

**Comment by bohua** [comment]
Hi Larry,

Unfortunately I didn't get any mails for that. could you verify if you have sent the snapshots to my mailbox?

Brs,
//Bohua

**Comment by ChryslerSupport** [comment]
Sorry, let me try again...
![issue](https://user-images.githubusercontent.com/23098200/28922457-72e46a20-7817-11e7-8c63-68f55e560cf8.PNG)

**Comment by bohua** [comment]
Hi Larry,

Could you also find the error request (https://shqlss01.server.chrysler.com:49...) in the network panel and snapshot the network panel details to me as well including the request and response:

<img width="651" alt="screen shot 2017-08-04 at 1 02 51 pm" src="https://user-images.githubusercontent.com/3418159/28966274-943971c6-7915-11e7-9465-0b281b750434.png">

By the way, this actually becomes more complicated than I expected. It will take long time by sending back&forth snapshots especially crossing timezone. :)  I would suggest to have a remote session if possible to resolve this issue quickly for your business. Of course, this will include a consultant service and related cost charged by my company (S-cubed). :)

Please let me know about your idea.

Thanks

//Bohua

**Comment by ChryslerSupport** [comment]
That is something we can set up next week.
We will talk about payment and charges.


On Aug 4, 2017, at 5:09 AM, bohua <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Larry,

Could you also find the error request (https://shqlss01.server.chrysler.com:49...) in the network panel and snapshot the network panel details to me as well including the request and response:

[screen shot 2017-08-04 at 1 02 51 pm]<https://user-images.githubusercontent.com/3418159/28966274-943971c6-7915-11e7-9465-0b281b750434.png>

By the way, this actually becomes more complicated than I expected. It will take long time by sending back&forth snapshots especially crossing timezone. :) I would suggest to have a remote session if possible to resolve this issue quickly for your business. Of course, this will include a consultant service and related cost charged by my company (S-cubed). :)

Please let me know about your idea.

Thanks

//Bohua

-

**Comment by bohua** [comment]
That's great, Larry! Please feel free to setup a meeting so we could talk about everything! 

Bohua Li

> 在 2017年8月4日，下午2:43，ChryslerSupport <notifications@github.com> 写道：
> 
> That is something we can set up next week.
> We will talk about payment and charges.
> 
>

**Comment by ChryslerSupport** [comment]
![ondemand2](https://user-images.githubusercontent.com/23098200/29092422-4947d260-7c54-11e7-9c65-a5ebd902583e.PNG)
![ondemand1](https://user-images.githubusercontent.com/23098200/29092421-493f3934-7c54-11e7-9f34-3f59de28d7ed.PNG)

Here is the traffic when a user clicks the button.

What time zone are you in?  I am Eastern Time UTC-5

**Comment by bohua** [comment]
Thanks Larry! I will have it investigated 

I am in Central European Time UTC+2.

Bohua Li

**Comment by ChryslerSupport** [comment]
Are you available to meet Monday, August 14th?  Unfortunately I don't have any availability this week.

**Comment by bohua** [comment]
Hi Larry I will have a trip from Friday until next Wednesday. Would be possible on next Thursday though?

Bohua Li

**Comment by ChryslerSupport** [comment]
yes, that sounds great,  Do you have an email address that you would like me to send a meeting request to?

**Comment by bohua** [comment]
Yes, please. bl@s-cubes.dk

Bohua Li

**Comment by ChryslerSupport** [comment]
Thant mail came back with this error....
Remote Server returned '< #5.4.4 smtp; 554 5.4.4 [internal] Domain Lookup Failed>'

**Comment by bohua** [comment]
Hi Larry,

Sorry! My mistake. 

bl@s-cubed.dk please. :)

Bohua Li

**Comment by jfortner03** [comment]
Did this issue get resolved? I believe we are having the same issue.

**Comment by bohua** [comment]
By configuring app permissions in NPrinting will resolve the issue
